test_datum__from_epsg fails with PROJ 9.4.1

#### Problem description

The test suite fails with PROJ 9.4.1:
```
118s =================================== FAILURES ===================================
118s ____________________________ test_datum__from_epsg _____________________________
118s 
118s     def test_datum__from_epsg():
118s         datum_wkt = (
118s             'ENSEMBLE["World Geodetic System 1984 ensemble",'
118s             'MEMBER["World Geodetic System 1984 (Transit)",ID["EPSG",1166]],'
118s             'MEMBER["World Geodetic System 1984 (G730)",ID["EPSG",1152]],'
118s             'MEMBER["World Geodetic System 1984 (G873)",ID["EPSG",1153]],'
118s             'MEMBER["World Geodetic System 1984 (G1150)",ID["EPSG",1154]],'
118s             'MEMBER["World Geodetic System 1984 (G1674)",ID["EPSG",1155]],'
118s             'MEMBER["World Geodetic System 1984 (G1762)",ID["EPSG",1156]],'
118s             'MEMBER["World Geodetic System 1984 (G2139)",ID["EPSG",1309]],'
118s             'ELLIPSOID["WGS 84",6378137,298.257223563,LENGTHUNIT["metre",1],'
118s             'ID["EPSG",7030]],ENSEMBLEACCURACY[2.0],ID["EPSG",6326]]'
118s         )
118s >       assert Datum.from_epsg("6326").to_wkt() == datum_wkt
118s E       assert 'ENSEMBLE["Wo..."EPSG",6326]]' == 'ENSEMBLE["Wo..."EPSG",6326]]'
118s E         
118s E         - ENSEMBLE["World Geodetic System 1984 ensemble",MEMBER["World Geodetic System 1984 (Transit)",ID["EPSG",1166]],MEMBER["World Geodetic System 1984 (G730)",ID["EPSG",1152]],MEMBER["World Geodetic System 1984 (G873)",ID["EPSG",1153]],MEMBER["World Geodetic System 1984 (G1150)",ID["EPSG",1154]],MEMBER["World Geodetic System 1984 (G1674)",ID["EPSG",1155]],MEMBER["World Geodetic System 1984 (G1762)",ID["EPSG",1156]],MEMBER["World Geodetic System 1984 (G2139)",ID["EPSG",1309]],ELLIPSOID["WGS 84",6378137,298.257223563,LENGTHUNIT["metre",1],ID["EPSG",7030]],ENSEMBLEACCURACY[2.0...
118s E         
118s E         ...Full output truncated (3 lines hidden), use '-vv' to show
118s 
118s test/crs/test_crs.py:730: AssertionError
118s =============================== warnings summary ===============================
```

https://ci.debian.net/packages/p/python-pyproj/unstable/amd64/47067339

#### Installation method

Debian packages

--- a/test/crs/test_crs.py
+++ b/test/crs/test_crs.py
@@ -728,7 +728,7 @@ def test_coordinate_operation__from_authority__empty():
 
 
 def test_datum__from_epsg():
-    datum_wkt = (
+    datum_wkt_prefix = (
         'ENSEMBLE["World Geodetic System 1984 ensemble",'
         'MEMBER["World Geodetic System 1984 (Transit)",ID["EPSG",1166]],'
         'MEMBER["World Geodetic System 1984 (G730)",ID["EPSG",1152]],'
@@ -737,10 +737,14 @@ def test_datum__from_epsg():
         'MEMBER["World Geodetic System 1984 (G1674)",ID["EPSG",1155]],'
         'MEMBER["World Geodetic System 1984 (G1762)",ID["EPSG",1156]],'
         'MEMBER["World Geodetic System 1984 (G2139)",ID["EPSG",1309]],'
+    )
+    datum_wkt_suffix = (
         'ELLIPSOID["WGS 84",6378137,298.257223563,LENGTHUNIT["metre",1],'
         'ID["EPSG",7030]],ENSEMBLEACCURACY[2.0],ID["EPSG",6326]]'
     )
-    assert Datum.from_epsg("6326").to_wkt() == datum_wkt
+    # Testing this way avoids problems when new members are added to the datum ensemble
+    assert Datum.from_epsg("6326").to_wkt().startswith(datum_wkt_prefix)
+    assert Datum.from_epsg("6326").to_wkt().endswith(datum_wkt_suffix)
 
 
 def test_datum__from_authority():

